go.mod

```diff
@@ -48,7 +48,7 @@ require (
 	github.com/filecoin-project/specs-actors/v2 v2.3.5
 	github.com/filecoin-project/specs-actors/v3 v3.1.1
 	github.com/filecoin-project/specs-actors/v4 v4.0.1
-	github.com/filecoin-project/specs-actors/v5 v5.0.1
+	github.com/filecoin-project/specs-actors/v5 v5.0.2-0.20210630111506-5d9e79a8e57e
```

Comment:
Can we get this into master / tagged?

Reply:
@magik6k that'll be a *very* long time...
https://github.com/filecoin-project/specs-actors/pull/1463#issuecomment-871475927

Reply:
@magik6k this now tracks current specs-actors master, I did not realize we can make the change there non-contentious.

Reply:
We can use https://github.com/filecoin-project/specs-actors/releases/tag/v5.0.3 now

Reply:
Thanks @ZenGround0, adjusted